proxysql/test-proxysql

```diff
@@ -38,45 +38,66 @@ function process_proxysql_output {
 }
 
 WORKDIR_ABS=$(cd ${1:-./build/proxysql}; pwd -P)
-rm -fr ${WORKDIR_ABS}/WORKDIR ${WORKDIR_ABS}/PXB ${WORKDIR_ABS}/PXC
-mkdir -p ${WORKDIR_ABS}/WORKDIR/PROXY  ${WORKDIR_ABS}/WORKDIR/PXC
+rm -rf ${WORKDIR_ABS}/WORKDIR ${WORKDIR_ABS}/PROXY_ADMIN
+mkdir -p ${WORKDIR_ABS}/PROXY_ADMIN ${WORKDIR_ABS}/WORKDIR/proxysql-2.0/usr/bin ${WORKDIR_ABS}/WORKDIR/proxysql-2.0/etc
 
 if [[ "${PXC_VERSION}" == "PXC80" ]]; then
-    sudo wget -q -O ${WORKDIR_ABS}/WORKDIR/Percona-XtraDB-Cluster_8.0.20-11.3_Linux.x86_64.glibc2.12-minimal.tar.gz https://www.percona.com/downloads/Percona-XtraDB-Cluster-LATEST/Percona-XtraDB-Cluster-8.0.20-11.3/binary/tarball/Percona-XtraDB-Cluster_8.0.20-11.3_Linux.x86_64.glibc2.12-minimal.tar.gz
-    sudo tar -C ${WORKDIR_ABS}/WORKDIR/PXC --strip-components=1 -zxpf $(ls $WORKDIR_ABS/WORKDIR/Percona-XtraDB-Cluster_8.0.20-11.3_Linux.x86_64.glibc2.12-minimal.tar.gz | head -1)
+# Downloading PXC-8.0 tarball package
+  sudo wget -q -O ${WORKDIR_ABS}/WORKDIR/Percona-XtraDB-Cluster_8.0.22-13.1_Linux.x86_64.glibc2.17-minimal.tar.gz https://www.percona.com/downloads/Percona-XtraDB-Cluster-LATEST/Percona-XtraDB-Cluster-8.0.22/binary/tarball/Percona-XtraDB-Cluster_8.0.22-13.1_Linux.x86_64.glibc2.17-minimal.tar.gz
+
+# Installing PXC-8.0 client package
+  sudo wget https://repo.percona.com/apt/percona-release_latest.generic_all.deb
+  sudo dpkg -i percona-release_latest.generic_all.deb
```

Comment:
in some cases it could fail.
We need to install gnupg and curl beforehand

Reply:
It's running inside docker container, in a pxc-build, where's the dependencies are installed before
* https://github.com/Percona-Lab/jenkins-pipelines/blob/master/pxc/docker/install-deps#L153

Reply:
in such case we also have percona-release installed

Reply:
Added installation steps for gnupg and curl beforehand